### PR TITLE
Android Sample Split

### PR DESCRIPTION
**Pull request by sbSteveK**
Move the Android sample to the samples folder and split up build of sdk and sample.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: documents/ANDROID.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+dependencies {
+    api 'software.amazon.awssdk.iotdevicesdk:aws-iot-device-sdk-android:1.17.0'
+}
+```
+Replace `1.17.0` in `software.amazon.awssdk.iotdevicesdk:aws-iot-device-sdk-android:1.17.0` with the latest release version for the SDK
+or replace with `1.0.0-SNAPSHOT` to use the SDK built and installed from source.
+Look up the latest SDK version here: https://github.com/aws/aws-iot-device-sdk-java-v2/releases
+
+## Samples App
+[Android IoT Samples App README](./samples/Android/README.md)
```
Comment (sfod) [inline documents/ANDROID.md:95]: I got 404 here

File: documents/ANDROID.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+* [Consuming IoT Device SDK Android](#consuming-from-locally-installed)
+  * [Consuming from Maven](#consuming-from-maven)
+  * [Consuming from locally installed](#consuming-from-locally-installed)
+* [Samples App](#samples-app)
+
+## Installation
+
+### Minimum requirements
+* Java 11+ ([Download and Install Java](https://www.java.com/en/download/help/download_options.html))
+  * [Set JAVA_HOME](./documents/PREREQUISITES.md#set-java_home)
```
Comment (sfod) [inline documents/ANDROID.md:36]: This returns 404

File: documents/ANDROID.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+* Gradle 7.4.2 ([Download and Install Gradle](https://gradle.org/install/))
+* Android SDK 26 ([Doanload SDK Manager](https://developer.android.com/tools/releases/platform-tools#downloads))
+  * [Set ANDROID_HOME](./documents/PREREQUISITES.md#set-android_home)
```
Comment (sfod) [inline documents/ANDROID.md:39]: This returns 404

File: documents/FAQ.md [hunk 4]
```diff
@@ -8,6 +8,7 @@
 * [Mac-Only TLS Behavior](#mac-only-tls-behavior)
 * [How do debug in VSCode?](#how-do-debug-in-vscode)
 * [What certificates do I need?](#what-certificates-do-i-need)
+* [How do I build and use the Android SDK](#how-do-i-build-and-use-the-android-sdk)
```
Comment (sfod) [inline documents/FAQ.md:11]: nit: Add `?`

File: samples/Android/README.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
+# Sample Application for the AWS IoT Device SDK Android
+[**Return to AWS IoT Device SDK for Android README**](../../documents/ANDROID.md)
+
+The Android sample builds an app that can be installed and run on an Android Device. The app builds and allows you
+to run the following [samples](#links-to-sample-readme) from aws-iot-device-sdk-java-v2:
```
Comment (sfod) [inline samples/Android/README.md:5]: Shouldn't it be `#links-to-individual-sample-readme-files`?

**Review by sfod — APPROVED** [review]

File: documents/ANDROID.md (hunk 1 above)
Comment (sbSteveK) [reply documents/ANDROID.md:95]: I'm getting the same. Best guess is they 404 until they've been added once? I'm pretty sure they're pointing to the correct location.

**Pull request merged**